src/commands/flinkArtifacts.test.ts

```diff
@@ -375,27 +355,24 @@ describe("flinkArtifacts", () => {
       .stub(CCloudResourceLoader.getInstance(), "getEnvironments")
       .resolves([mockEnvironment as CCloudEnvironment]);
 
-    const responseError = {
-      name: "ResponseError",
-      message: "Failed to create UDF function: Malformed JSON",
-      isResponseError: true,
-      response: {
-        status: 400,
-        statusText: "Bad Request",
-        headers: new Headers(),
-        // This is an object that will be serialized
-        body: { malformedJson: true },
-      },
-      statusCode: 400,
-    };
+    const responseForNewError = createResponseError(400, "Bad Request", {
+      "Malformed JSON": true,
+    } as any);
```

Comment:
here, had to coerce the parameter to insert the malformed JSON, otherwise with a good param, we'd just be testing regular response, not malformed